test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignerTests.cs

```diff
@@ -88,27 +88,25 @@ public SignerTests(SigningTestFixture fixture)
             using (var test = new Test(_testFixture.TrustedTestCertificateNotYetValid.Source.Cert))
             {
                 var exception = await Assert.ThrowsAsync<SignatureException>(
-                    () => test.Signer.SignAsync(test.Request, NullLogger.Instance, CancellationToken.None));
+                    () => test.Signer.SignAsync(NullLogger.Instance, CancellationToken.None));
 
                 Assert.Equal(NuGetLogCode.NU3017, exception.Code);
                 Assert.Contains("The signing certificate is not yet valid", exception.Message);
 
-                var isSigned = await IsSignedAsync(test.WriteStream);
+                var isSigned = await IsSignedAsync(test.Request.OutputPath);
 
                 Assert.False(isSigned);
             }
         }
 
-        private static async Task<bool> IsSignedAsync(MemoryStream package)
+        private static async Task<bool> IsSignedAsync(string packagePath)
         {
-            var currentPosition = package.Position;
+            var package = File.OpenRead(packagePath);
```

Comment:
`File.OpenRead(...)` returns a stream, which implements `IDisposable`.  You must wrap this in a `using` statement.